__tests__/formatter/__snapshots__/javascript.test.ts.snap

```diff
@@ -1,51 +1,41 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Synthetics JavaScript formatter inline journeys 1`] = `
-"step('Go to https://vigneshh.in/', async () => {
-  await page.goto('https://vigneshh.in/');
-  expect(await page.isVisible('text=Babel Minify')).toBeTruthy();
-  expect(await page.isEditable('text=Babel Minify')).toBeTruthy();
-  expect(await page.textContent('text=Babel Minify')).toMatch('Babel');
-});
-step('Click text=Babel Minify', async () => {
-  const [page1] = await Promise.all([
-    page.waitForEvent('popup'),
-    page.click('text=Babel Minify')
-  ]);
-  await Promise.all([
-    page1.waitForNavigation(/*{ url: 'https://github.com/babel/minify' }*/),
-    page1.click('a:has-text(\\"smoke\\")')
-  ]);
-  await page1.close();
-});"
-`;
-
-exports[`Synthetics JavaScript formatter suite journeys 1`] = `
-"const { journey, step, expect } = require('@elastic/synthetics');
+exports[`Synthetics JavaScript formatter accepts custom step organization 1`] = `
+Array [
+  "const { journey, step, expect } = require('@elastic/synthetics');
```

Comment:
the formatting if not correct.

Reply:
Overwrote this in https://github.com/elastic/synthetics/pull/457/commits/069aaba4c69b93887ba4b83f18a438031926a709.

It's now a snapshot of a single string that looks ok.